mobile-app/app/screens/PlaygroundNavigator/sections/PlaygroundConnection.tsx

```diff
@@ -1,60 +1,22 @@
 
-import { useSelector } from 'react-redux'
-import { Text, View } from '../../../components'
-import { ThemedText, ThemedView } from '@components/themed'
+import { ThemedTextV2, ThemedViewV2 } from '@components/themed'
 import { useNetworkContext } from '@shared-contexts/NetworkContext'
 import { isPlayground } from '@environment'
-import { RootState } from '@store'
 import { tailwind } from '@tailwind'
-import { PlaygroundTitle } from '../components/PlaygroundTitle'
 
 export function PlaygroundConnection (): JSX.Element {
   const { network } = useNetworkContext()
-  const connected = useSelector((state: RootState) => state.block.connected)
-  const count = useSelector((state: RootState) => state.block.count)
 
   return (
-    <View>
-      <PlaygroundTitle
-        status={{ online: connected, offline: !connected }}
-        title='Connection'
-      />
+    <ThemedViewV2 style={tailwind('px-5 mt-8 mb-6')}>
+      {
+        isPlayground(network) && (
+          <ThemedTextV2 style={tailwind('text-sm')}>
+            DeFi Playground is a testing blockchain isolated from MainNet for testing DeFi applications with mock assets.
+          </ThemedTextV2>
+        )
+      }
+    </ThemedViewV2>
 
-      <ThemedView
-        dark={tailwind('bg-gray-800')}
-        light={tailwind('bg-white')}
-        style={tailwind('px-4 py-4')}
-      >
-        <Text>
-          <ThemedText style={tailwind('font-medium')}>
-            Network:
-          </ThemedText>
-
-          <ThemedText
-            testID='playground_active_network'
-          >
-            {' '}
-
-            {network}
-          </ThemedText>
-        </Text>
-
-        <ThemedText>
-          Blocks:
-          {' '}
-          {count === 0 ? '...' : <Text testID='current_block_count'>{count}</Text>}
```

Comment:
![image](https://user-images.githubusercontent.com/13292662/185824897-0979a5d0-07c6-4fd6-9a58-3ee523e2ecfd.png)
Failing test due to this

Reply:
done